src/sonic_ax_impl/mibs/__init__.py

```diff
@@ -10,6 +10,8 @@
 from ax_interface.util import oid2tuple
 from sonic_ax_impl import logger
 
+from .physical_entity_sub_oid_generator import *
```

Comment:
>physical_entity_sub_oid_generator [](start = 6, length = 33)

This file is only used by rfc2737.py. You may move code to
1. src/sonic_ax_impl/lib/
2. or, src/sonic_ax_impl/mibs/ietf
And only import in required file.